fastchat/serve/monitor/classify/category.py

```diff
@@ -26,14 +36,110 @@ def create_category(name):
             return CategoryMath()
         elif name == "creative_writing_v0.1":
             return CategoryCreativeWriting()
+        elif name == "refusal_v0.2":
```

Comment:
Now I think about there is no use for the class `Category` since all this is doing is call the function `create_category(name)`, so why not just turn it into a function instead of making a parent class. This way we will have two parent classes `CategoryAPI` and `CategoryHF`. Because right now there is like a 3 level pyramid architecture which is kinda too much I think.